packages/gatsby-theme/src/blocks/ShareBlock/index.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+import Share from "../../components/Share"
```

Comment:
Pleas update this import
```suggestion
import { Share } from "ui"
```